invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LocationList/LocationList.js

```diff
@@ -68,13 +63,7 @@ export default class LocationList extends Component {
 
   renderResults(data) {
     const rows = this.prepareData(data);
-    const headerActionComponent = (
-      <NewButton
-        clickHandler={() => {
-          openRecordEditor(locationApi.url);
```

Comment:
@ntarocco I would propose to just make the `openRecordEditor` function dummy so after we implement the forms to be able to find easily all the buttons needed to point to the corresponding form...WDYT?

Reply:
I will change as suggested by adding a comment!